conversation_reconstruction_dataflow/conversation_reconstruction/construct_utils/conversation_constructor.py

```diff
@@ -217,16 +220,25 @@ def insert(rev, page, previous_comments, DEBUGGING_MODE = False):
     eof = max(list(updated_page['actions'].keys()))
     if DEBUGGING_MODE:
        print(eof)
+    for action, val in updated_page['actions'].items():
+        if not(action == eof):
+           assert not(val == (-1, -1)) 
     assert updated_page['actions'][eof] == (-1, -1)
     
+    updated_actions = sorted(updated_actions, key = lambda k: int(k['id'].split('.')[1]))
     return updated_actions, updated_page
 
 
 class Conversation_Constructor:
     def __init__(self):
         self.page = {}
-        self.THERESHOLD = 3 # A comment with at least THERESHOLD number of tokens will be recorded 
+        self.THERESHOLD = 10 # A comment with at least THERESHOLD number of tokens will be recorded 
```

Comment:
Comment maybe clearer as: "comments with less than this number of tokens will be skipped" (because the expectation is already that we are keeping adding comments to ingestion :) 

(curious how many comments are like this? why not keep all comments?)

Reply:
This is more like: Deleted comments with less than this number of tokens will not be recorded thus not considered in comment restoration actions. 

It was set this way to reduce confusion of restoration actions. Short comments like 'Thank you' can be part of other comments, say: "Thank you so much for your effort." The idea of not including deleted version of 'Thank you' is that we want to identify when adding text like  "Thank you so much for your effort.", it's an independent insertion of new comment instead of partial restoration of 'Thank you'.

Reply:
I see, that makes sense, lets write that in the comment.